sdks/how-to/how-to-migrate-to-icure-premium-features.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: how-to-migrate-to-icure-premium-features
+description: How to Migrate to iCure Premium Features
+tags:
+- Open Source
+- Migration
+- Premium Features
+---
+
+# How to Migrate to iCure Premium Features
+
+:::note
+
+This how-to assumes that you have been using the iCure Medical Device SDK in your application with the open-source version
+ of the iCure backend and that, after creating an application, a database, and a suer using the iCure Cockpit, you now
```

Comment:
`This how-to assumes that until now, you have been using the iCure Medical Device SDK in your application with the open-source version of iCure.  You now want to start using the iCure Cloud version, including all premium features.`